src/corehost/build.sh

```diff
@@ -184,6 +184,13 @@ else
     exit 1
 fi
 
+if [ -z $CONFIGURATION ]; then
+    export CONFIGURATION=Debug
+fi
+echo "CONFIGURATION=$CONFIGURATION"
+
+__cmake_defines="-DCMAKE_BUILD_TYPE=${CONFIGURATION} ${__cmake_defines}"
```

Comment:
@hqueue I fixed this in https://github.com/dotnet/core-setup/commit/00ee96f8a542a90fa1cf8bf9f9beebae1592e383#diff-84bd659d6cb7857c6f9d4f557e88d950. I presume you were using this to fix https://github.com/dotnet/core-setup/issues/1386 - if so, is there any other change you were planning to bring in?